### PR TITLE
Top ad below nav ab test

### PR DESCRIPTION
**Pull request by Calanthe**
Ab test to check if top banner below nav instead of above, will increase viewability.

screen shot 2015-03-17 at 16 57 33

And yes, jumps are sometimes noticeable, but it's only ab test and we didn't find any easy solution.

### CODE REVIEW

File: common/app/conf/switches.scala [hunk 1]
```diff
@@ -471,6 +471,11 @@ object Switches {
     safeState = Off, sellByDate = new LocalDate(2015, 5, 11)
   )
 
+  val ABMtTopBelowNav = Switch("A/B Tests", "ab-mt-top-below-nav",
```
Comment (kelvin-chappell) [inline common/app/conf/switches.scala]: Where is this switch used?

Comment (Calanthe) [inline common/app/conf/switches.scala]: It is used in the switchboard eg here https://frontend.code.dev-gutools.co.uk/dev/switchboard

Comment (kelvin-chappell) [inline common/app/conf/switches.scala]: It looks like it's also being used as part of the test framework here: https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/ab.js#L82

**Comment by kelvin-chappell** [comment]
:+1:

**Pull request merged**